src/main/java/com/tahera/model/Book.java

```diff
@@ -1,9 +1,17 @@
 package com.tahera.model;
 
-import javax.persistence.*;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
```

Comment:
same concern here

Reply:
Again, these were automatically handlede by the ide, but avoids using the wildcard just for a few dependencies.